File: python/person.py
import pickle

class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age

    def __str__(self):
        return f"Person(name={self.name}, age={self.age})"

# Create an instance of the Person class
person = Person("John Doe", 30)
serialized_person = pickle.dumps(person)
print(serialized_person)

File: php/person.php
<?php
class Person {
    public $name;
    protected $age;
    private $password;

    public function __construct($name, $age, $password) {
        $this->name = $name;
        $this->age = $age;
        $this->password = $password;
    }

    public function display() {
        echo "Name: {$this->name}, Age: {$this->age}";
    }
}

$person = new Person("John Doe", 30, "secret");

$serialied_person = serialize($person);
echo 'Serialized valuue: ' . $serialied_person;
?>
